style.css

```diff
@@ -0,0 +1,106 @@
+
+
```

Comment:
Please be careful with blank lines and indentation. refer to _HTML & CSS best practices_

Reply:
- [ ] Being careful with blank lines is not very unclear. **What do you want the student to do about them**? I suggest you state the comment as **Please be consistent in your coding style and avoid leaving blank lines within a code block. You can skip one blank line between code blocks**. This feature can as well be enforced by linters so you could also advise the student to use linters to enforce this rule 👍🏾 

- [ OPTIONAL] Instead of saying *refer to HTML & CSS best practices*, you could instead add a [link to the best practice ](refer to HTML & CSS best practices) article at microverse or any other authentic source 😄